woocommerce/woocommerce-android
# [Mobile Payments] Do use full screen input for simple payments amount input

Closes: #8382 


### Description

it seems that the keyboard owns the fullscreen mode and for some reason, it doesn't show separators, so here I implemented a workaround that makes it to not use a fullscreen mode. That's not ideal though

### Testing instructions

* Start Simple Payments Flow
* On the Amount Entering screen move to landscape mode
* Notice that you can enter the amount normally

### Images/gif


![screencap-2023-03-09T105640 781Z](https://user-images.githubusercontent.com/4923871/224003522-a2814404-79ad-4aca-81c9-450a75c7d80c.png)
